### PR TITLE
Auditcare migration changelog

### PR DESCRIPTION
**Pull request by AmitPhulera**
Adds a Changelog for auditcare historical data migration
##### ENVIRONMENTS AFFECTED

Any env that uses auditcare.

### CODE REVIEW

**Review by millerdev — APPROVED** [review]

File: changelog/0051-historical-auditcare-data-migration.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  ### Steps to migrate
+  It is advised to start a tmux session to run the migrations, you can use `django-manage` machine to start the tmux session. 
+    
+  ```
+  $ cchq <env> ssh django-manage:0
+  $ tmux new -s auditcare
+  $ sudo -iu cchq
+  $ cd /home/cchq/www/<env>/current && source python_env/bin/activate
+  $ python manage.py copy_events_to_sql --batch_size=10000 --workers=5
+  ```
```
Comment (dannyroberts) [inline changelog/0051-historical-auditcare-data-migration.yml:21]: Any reason not to use `django-manage --tmux copy_events_to_sql ...`?

Comment (AmitPhulera) [reply changelog/0051-historical-auditcare-data-migration.yml:21]: I like to give the tmux shell a recognizable name while creating it, it's handy to attach back. It's just a personal preference, I can go with `djnago-manage` if you feel that would be more apt.

**Comment by AmitPhulera** [comment]
Hey @dannyroberts, 
Do you have any insight on test failure here?

```
If you're getting a failure here with a change to the changelog docs
    make sure you're following the directions in
    https://github.com/dimagi/commcare-cloud/blob/master/changelog/README.md
    and not editing changelog markdown files directly.
```

I don't think I have edited any markdown files manually.

**Comment by dannyroberts** [comment]
The part of the error message you pasted doesn't apply in this case (i.e. you're not "getting a failure here with a change to the changelog docs"). It's complaining because you seem to have deleted `hosting_docs/source/reference/1-commcare-cloud/commands.md` and it expects that file to be there. (You can see the diff above is basically "our version of what is here includes this entire file that isn't present".)

File: changelog/0051-historical-auditcare-data-migration.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  Replace <env> with your correct environemt
```
Comment (millerdev) [inline changelog/0051-historical-auditcare-data-migration.yml:22]:
```suggestion
  Replace <env> with your correct environment
```

**Pull request merged**